docs/source/develop.rst

```diff
@@ -7,30 +7,30 @@ and documentation and style standards are available at the `Dask developer
 guidelines`_ in the main documentation.
 
 .. _Dask: http://dask.org
-.. _`Dask developer guidelines`: http://docs.dask.org/en/latest/develop.html
```

Comment:
Do we want to add this back in?  There is a broken link just above otherwise.

Reply:
that's a link back to the very same document, it didn't make any sense to me

Reply:
what do you mean with broken link? http://dask.org works just fine

Reply:
Across lines 6-7 above there's a reference to 

```
`Dask developer guidelines`_
```

You are removing the target, so that link will break.

Reply:
Ah I see it now